data/sql/updates/pending_db_world/rev_1617265345631987700.sql

```diff
@@ -12,7 +12,7 @@ INSERT INTO `waypoints` VALUES
 
 DELETE FROM `smart_scripts` WHERE (`entryorguid` = 398400) AND (`source_type` = 9)
 
-UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE (`entry` IN (3984, 3985));
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE (entry IN (3984, 3985));
```

Comment:
why?

Reply:
because it gave me an error, and i tried like this..

Reply:
in my DB it work in both ways

Reply:
idk why it gave this error

Reply:
That's weeeeeird, ok ^^

Reply:
i putted it like before.. give a look